certbot-dns-route53/certbot_dns_route53/dns_route53.py

```diff
@@ -18,6 +18,7 @@
     "https://boto3.readthedocs.io/en/latest/guide/configuration.html#best-practices-for-configuring-credentials "  # pylint: disable=line-too-long
     "and add the necessary permissions for Route53 access.")
 
+# pylint: disable=abstract-method
```

Comment:
Maybe define a dummy `_perform` similar to `_setup_credentials` instead of suppressing the warnings? @bmw  pointed out that if we ever decide to have `common.Plugin` properly extend `ABCMeta`, then [we wouldn't be able to instantiate this class](https://docs.python.org/3/library/abc.html#abc.abstractmethod).

Reply:
Done. I feel a little weird about private methods being abstract, now that I think about it; I'll pick some brains in person when I get a chance.